aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -1275,6 +1278,8 @@ void onServiceAck(
                     {
                         ClusterControl.ToggleState.reset(controlToggle);
                     }
+
+                    snapshotDurationTracker.onSnapshotEnd(clusterClock.timeNanos());
```

Comment:
This will miss the case when snapshot is taken before the Cluster termination, i.e. in which case the snapshot time will not be recorded. Maybe move it into the `takeSnapshot` method.